new-log-viewer/src/utils/downloadDecompressedLogs.ts

```diff
@@ -0,0 +1,23 @@
+interface downloadDecompressedLogsProps {
+    blob: Blob,
+    fileName: string,
+}
+
+/**
+ *
+ * @param blob.blob
+ * @param blob
+ * @param fileName
+ * @param blob.fileName
+ */
+const downloadDecompressedLogs = ({blob, fileName}: downloadDecompressedLogsProps) => {
```

Comment:
Shall we rename it as `downloadBlob` or `exportBlob`?

Reply:
Let's use `downloadBlob`